tests/js_tests/spec/VCF.spec.js

```diff
@@ -10,9 +10,9 @@ require([
         ) {
 describe('VCF store', function() {
 
-  
 
-  it('reads big dbsnp', function() {
+
+  xit('reads big dbsnp', function() {
```

Comment:
don't forget to remove this x

Reply:
I don't think this file exists anywhere, so it is unable to pass by itself? if it is large maybe it should not be added to repo after all

Reply:
oh, yeah it doesn't exist.  how were these tests passing running in Travis then??

Reply:
I don't think they were running currently index.html has vcf.spec.js commented this PR uncommented it but xit that test

Reply:
oh god thanks for catching that